crates/data_components/src/flightsql.rs

```diff
@@ -54,26 +54,35 @@ pub mod federation;
 
 #[derive(Debug, Snafu)]
 pub enum Error {
-    #[snafu(display("Unable to connect to FlightSQL Server"))]
+    #[snafu(display("Failed to connect to the Flight server.\n{source}\nVerify configuration and try again. For details, visit https://docs.spiceai.org/components/data-connectors/flightsql#params"))]
     UnableToConnectToServer { source: tonic::transport::Error },
 
-    #[snafu(display("Unable to generate SQL: {source}"))]
+    #[snafu(display("Failed to create SQL query (flightsql).\n{source}\nAn unexpected error occurred. Please report a bug on GitHub: https://github.com/spiceai/spiceai/issues"))]
```

Comment:
lgtm, we should probably make the `(flightsql)` a pattern (i.e. mention the context information for the same errors that might happen for different connector / accelerator).